examples/main.go

```diff
@@ -67,4 +68,24 @@ func main() {
 		client.WithBatchEventProcessor(event.DefaultBatchSize, event.DefaultEventQueueSize, event.DefaultEventFlushInterval),
 	)
 	optimizelyClient.Close()
+
+	/************* Setting experiment overrides (a.k.a. "forced variations") ********************/
+	overrideKey := decision.OverrideKey{
+		Experiment: "aaaa",
+		User:       "Matt",
+	}
+	overrides := map[decision.OverrideKey]string{
+		overrideKey: "variation_1",
+	}
+	compositeService := decision.NewCompositeService(
+		sdkKey,
+		decision.WithExperimentOverrides(overrides),
+	)
+	optimizelyClient, _ = optimizelyFactory.Client(
+		client.WithDecisionService(compositeService),
```

Comment:
Since overrides are a first-party feature, I think it would be good to have an option func directly on the client factory to instantiate a composite service with the overrides instead of having to rely on instantiating a compositeService outside of the client.